### DISCUSSION TITLE
Python api output is incomplete

### DISCUSSION THREAD

**Discussion by hassen-miri** [osm-search/Nominatim#3614] [Q&A]
I'm trying to use Nominatim python library for reverse geocoding, I see that the format is less precise than the output of HTTP result, I use the following code

```
lon = -123.148978
lat = 45.198726
coord = (lon, lat)  # Pass coordinates as a tuple (lon, lat)
result =  await api.reverse(coord)  # Perform reverse geocoding with the tuple
```
to get the following result

`{'tiger:county': 'Yamhill, OR'}`
whereas on the API I get the following result

```
{'road': 'Southeast Three Mile Lane',
 'town': 'McMinnville',
 'county': 'Yamhill County',
 'state': 'Oregon',
 'ISO3166-2-lvl4': 'US-OR',
 'postcode': '97128',
 'country': 'United States',
 'country_code': 'us'}
```
I am new to the tool, any idea on how to use this or how to find detailed documentation on how to use these functions ?

**Comment by mtmail** [comment]
https://nominatim.org/release-docs/latest/library/Getting-Started/

https://nominatim.org/release-docs/latest/library/NominatimAPI/#nominatimapi

Can you post the full result? Isn't there a 'osm_type', 'name', 'license' field, too?

**Comment by lonvia** [comment]
You need to enable output of addressdetails, like so:

```
result =  await api.reverse(coord, address_details=True)
```

**Reply by hassen-miri** [reply]
thanks !
